### PR TITLE
Use a separate directory for reports

### PR DESCRIPTION
**Pull request by chentex**
Signed-off-by: Vicente Zepeda Mas <vzepedam@redhat.com>

### Description

This will make easier to upload all the results to an Elasticsearch instance or something else

### CODE REVIEW

File: ocm-api-load.example.yaml [hunk 1]
```diff
@@ -19,6 +19,9 @@ tests:
   register-new-cluster:
     freq: 1000
     per: "h"
+  register-existing-cluster:
```
Comment (kurtismullins) [inline ocm-api-load.example.yaml:22]: Oops; I overlooked this file in my previous PR and had only added it to the readme. Thank you!

File: pkg/helpers/types.go [hunk 2]
```diff
@@ -28,5 +28,7 @@ type TestOptions struct {
 	Metrics         map[string]*vegeta.Metrics     // Stores results from each Test
 	Attacker        *vegeta.Attacker
 	OutputDirectory string // Directory to write vegeta results as a File
-	Connection      *sdk.Connection
+	// TODO make cofigurable???
```
Comment (kurtismullins) [inline pkg/helpers/types.go:31]: It would not be a bad idea but not super important for now. We can always add that in later.

**Review by kurtismullins — APPROVED** [review]
LGTM

File: ocm-api-load.example.yaml (hunk 1 above)
Comment (chentex) [reply ocm-api-load.example.yaml:22]: Don't worry, I didn't notice also

**Pull request merged**